hugo.toml

```diff
@@ -517,3 +516,74 @@ languagedirection = "ltr"
 languageNameLatinScript = "Tiếng Việt"
 description = "Giải pháp điều phối container trong môi trường production"
 
```

Comment:
We should add a comment here, I think.
What happens if someone creates, eg `i18n/de/en.toml`?

The comment I have in mind warns people not to try that.

Reply:
Sounds good! Done!